concord-server/server/src/main/resources/migrations.xml

```diff
@@ -87,4 +87,10 @@
             label = 'TRASH'
         </sql>
     </changeSet>
+    <changeSet id="6" author="wilmol">
+        <comment>Users should not have duplicate votes for a given phrase</comment>
+        <sql dbms="mysql">
+            ALTER TABLE votes DROP PRIMARY KEY , ADD PRIMARY KEY ( phraseId, userId )
```

Comment:
do we need a query to delete the votes violating the new PK...

Reply:
I'd say in an ideal world, yes, but to be practical probably not given no one else is using concord yet